### PR TITLE
media-gfx/gpicview: update GTK+ icon cache and xdg desktop database

### PR DESCRIPTION
**Pull request by stha09**
* fix QA warnings

### CODE REVIEW

File: media-gfx/gpicview/gpicview-0.2.5-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	>=x11-libs/gtk+-2.6:2"
+DEPEND="${RDEPEND}
+	>=dev-util/intltool-0.40
+	virtual/pkgconfig
+	sys-devel/gettext"
+
+src_install() {
+	emake DESTDIR="${D}" install
+	dodoc AUTHORS
+}
```
Comment (titanofold) [inline media-gfx/gpicview/gpicview-0.2.5-r1.ebuild]: src_install is unnecessary. It doesn't deviate from the default. Go ahead and delete it.

File: media-gfx/gpicview/gpicview-0.2.5-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+inherit gnome2-utils xdg-utils
+
+DESCRIPTION="A Simple and Fast Image Viewer for X"
+HOMEPAGE="http://lxde.sourceforge.net/gpicview"
+SRC_URI="mirror://sourceforge/lxde/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ppc ~x86 ~arm-linux ~x86-linux"
+IUSE=""
```
Comment (titanofold) [inline media-gfx/gpicview/gpicview-0.2.5-r1.ebuild]: Delete empty variables.

File: media-gfx/gpicview/gpicview-0.2.5-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+RDEPEND="virtual/jpeg:0
+	>=x11-libs/gtk+-2.6:2"
+DEPEND="${RDEPEND}
+	>=dev-util/intltool-0.40
+	virtual/pkgconfig
+	sys-devel/gettext"
```
Comment (titanofold) [inline media-gfx/gpicview/gpicview-0.2.5-r1.ebuild]: Alphabetical, please, grouping use flag constraint, version constraint, and bare depends together in the same order for each *DEPEND var.

**Comment by stha09** [comment]
Thanks for review, hope it didn't miss something.

**Pull request closed without merging**